README.md

```diff
@@ -5,6 +5,12 @@ The main purpose of this module is to provide a PuppetDB import source
 for Icinga Director. As of this writing it has not even a GUI or CLI
 component, but this may change in future.
 
+To install, please refer to our documentation. [Installation and configuration](doc/01-Installation.md)
+Note: Some steps require you to be familiar with Puppet/PuppetDB. 
+
+
+==========
+
+Screnshot:
```

Comment:
Remove the line with the == signs, as that generates a h1 with no text. Underline Screenshot (typo, e VS ee) with dashes ( -- ) and remove the colon.